From af4da2631ea9404dd15384d47f2949701b51868c Mon Sep 17 00:00:00 2001
From: Rashed Abdel-Tawab <rashed@linux.com>
Date: Sun, 15 Apr 2018 10:18:26 -0700
Subject: [PATCH] libc: Mark libstdc++ as vendor available

A lot of qualcomm blobs still link this even on 8.1,
so allow these deviced to build a vendor copy of it.

Change-Id: I2349478ec0507e3a5136fe89f15e7dc4bfc1a03e
---
 libc/Android.bp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/libc/Android.bp b/libc/Android.bp
index 6717095b9..ef15d91a8 100644
--- a/libc/Android.bp
+++ b/libc/Android.bp
@@ -1871,7 +1871,8 @@ cc_library {
     ],
     name: "libstdc++",
     system_shared_libs: ["libc"],
-		static_libs: ["libasync_safe"],
+    static_libs: ["libasync_safe"],
+    vendor_available: true,
 
     //TODO (dimitry): This is to work around b/24465209. Remove after root cause is fixed
     arch: {
-- 
2.17.0

